manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -92,7 +92,7 @@ spec:
       sourceLabels:
       - __name__
     - action: drop
-      regex: apiserver_request_duration_seconds_bucket;(0.15|0.25|0.3|0.35|0.4|0.45|0.6|0.7|0.8|0.9|1.25|1.5|1.75|2.5|3|3.5|4.5|6|7|8|9|15|25|30|50)
+      regex: apiserver_request_duration_seconds_bucket;(0.4|0.6|0.8|1.25|1.5|3|6|8|15)
```

Comment:
I removed some of the buckets that were dropped but aren't in the 1.23 list of buckets (https://github.com/kubernetes/kubernetes/pull/106306/commits/a8bd20a5af1174ce9f9a185b6d49f025a1045b03) and I also remove the 30s bucket since it is now used in the alert.